db/schema.rb

```diff
@@ -356,6 +360,19 @@
     t.index ["source_id"], name: "index_permission_templates_on_source_id", unique: true
   end
 
+  create_table "permits", id: :integer, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
I think the db migrations are messed up on your development machine.

Reply:
I'm gonna delete this PR and rebuild my heliotrope dir from scratch. Everything is hosed now.